src/main/java/org/stellar/sdk/responses/LiquidityPoolResponse.java

```diff
@@ -0,0 +1,149 @@
+package org.stellar.sdk.responses;
+
+import com.google.common.base.Objects;
+import com.google.gson.annotations.SerializedName;
+
+import org.stellar.sdk.Asset;
+import org.stellar.sdk.LiquidityPoolID;
+import org.stellar.sdk.xdr.LiquidityPoolType;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.errorprone.annotations.ForOverride;
+
+/**
+ * Represents liquidity pool response.
+ * @see <a href="https://www.stellar.org/developers/horizon/reference/resources/liquiditypool.html" target="_blank">Liquidity Pool documentation</a>
+ * @see org.stellar.sdk.requests.LiquidityPoolsRequestBuilder
+ * @see org.stellar.sdk.Server#liquidityPools()
+ */
+public class LiquidityPoolResponse extends Response {
+  @SerializedName("id")
+  private LiquidityPoolID id;
+  @SerializedName("paging_token")
+  private String pagingToken;
+  @SerializedName("fee_bp")
+  private Integer feeBP;
+  @SerializedName("type")
+  private LiquidityPoolType type;
+  @SerializedName("total_trustlines")
+  private String totalTrustlines;
```

Comment:
you can use Longs instead of Strings because the java json parser will deserialize the string properly into a long